Automattic/wp-calypso
# Stats Widget: Fix dashboard styles impacted by widget styling

Related to #75360

## Proposed Changes

* Extract modified layout styles only for the Odyssey Stats page to an independent file.
* Use minimal base styles built from Calypso to reduce the impact on base elements, e.g., `<p>` and `<ul>`.

|Before|After|
|-|-|
|<img width="1916" alt="截圖 2023-04-18 下午9 42 05" src="https://user-images.githubusercontent.com/6869813/232798230-bb3cc997-f809-4054-88a1-8e6a33b4ec4e.png">|<img width="1927" alt="截圖 2023-04-18 下午9 41 12" src="https://user-images.githubusercontent.com/6869813/232798302-e5589939-542f-46e3-b629-1368155ea8cf.png">|

## Testing Instructions



* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php?odyssey_widget=1`.
* Ensure the `Dashboard` page styling works the same as the page without the feature flag `odyssey_widget=1`.

## Pre-merge Checklist
